src/actions/contact-request/saveContactRequest.ts

```diff
@@ -0,0 +1,21 @@
+'use server'
+import { createContactRequest } from '@/backend/contact-request/contact-request.service'
+import { ContactFormRequest } from '@/components/ContactForm/schema'
+
+export const saveContactRequest = async (
+  contactFormRequestData: ContactFormRequest,
+) => {
+  try {
+    const createdContactRequest = await createContactRequest(
+      contactFormRequestData,
+    )
+    if (createdContactRequest) {
+      createdContactRequest
```

Comment:
**nit:** it's missing return, and let's remove booleans as discussed on other PR